src/SFA.DAS.Provider.PR.Web.UnitTests/Validators/SearchByPayeSubmitViewModelValidatorTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using FluentValidation.TestHelper;
+using SFA.DAS.Provider.PR.Web.Models.AddEmployer;
+using SFA.DAS.Provider.PR.Web.Validators;
+
+namespace SFA.DAS.Provider.PR_Web.UnitTests.Validators;
+public class SearchByPayeSubmitViewModelValidatorTests
+{
+    [TestCase("111/1", "1234567890123")]
+    [TestCase("111/12", "1234567890123")]
+    [TestCase("111/ABC", "1234567890123")]
+    [TestCase("111/ABCD", "1234567890123")]
+    [TestCase("111/ABCDE", "1234567890123")]
+    [TestCase("111/ABCDEF", "1234567890123")]
+    [TestCase("111/ABCDEFG", "1234567890123")]
+    [TestCase("111/ABCDEFGH", "1234567890123")]
+    [TestCase("111/ABCDEFGHI", "1234567890123")]
+    [TestCase("111/ABCDEFGHIJ", "1234567890123")]
+    public void ValidDetailsInModel_IsValid(string payeRef, string aorn)
```

Comment:
I got confused on the intension of the test. This seems like testing valid scenario for PAYE as AORN is constant here. I would just write this test specifically for valid PAYE cases. Also I do not see point of incrementing test for valid length, border line tests would suffice.

Reply:
ok